### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by ciremoussadia**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,32 +1,26 @@
 source "https://rubygems.org"
 
-ruby '1.9.3'
+ruby '2.7.2'
```
Comment (blambeau) [inline Gemfile:3]: Remove ruby version from Gemfile.

File: Gemfile [hunk 2]
```diff
@@ -1,32 +1,26 @@
...
-gem "alf-core",   :git => "git://github.com/alf-tool/alf-core.git"
-gem "alf-doc",    :git => "git://github.com/alf-tool/alf-doc.git"
-gem "alf-sql",    :git => "git://github.com/alf-tool/alf-sql.git"
-gem "alf-sequel", :git => "git://github.com/alf-tool/alf-sequel.git"
-gem "alf-rack",   :git => "git://github.com/alf-tool/alf-rack.git"
+gem "alf-core",   "~> 0.16.3"
+gem "alf-doc",    "~> 0.16.3"
+gem "alf-sql",    "~> 0.16.3"
+gem "alf-sequel", "~> 0.16.3"
+gem "alf-rack",   "~> 0.16.3"
```
Comment (blambeau) [inline Gemfile:19]: Ok, but are we sure there are no non-commited changes on master of those git repos?

Comment (ciremoussadia) [reply Gemfile:19]: @blambeau I thought it was preferable to fix the version so not to be surprised by new changes?

Comment (blambeau) [reply Gemfile:19]: Yeah, but I would not be surprised that try-alf uses non released features. Could you maybe check (on alf-core, alf-rack and alf-sql) whether there are unreleased changes (compare v0.16.3..master and see how many commits)?

Comment (ciremoussadia) [reply Gemfile:19]: Ok

Comment (ciremoussadia) [reply Gemfile:19]: @blambeau 21 commits.

**Comment by blambeau** [comment]
Keep moving with your change and see if try-alf can start.

I'll release 0.16.4 if needed.

B

On January 12, 2021, GitHub <notifications@github.com> wrote:
> @ciremoussadia commented on this pull request.
>

> In Gemfile <https://github.com/alf-tool/try-
> alf/pull/3#discussion_r555895254>:
>

> gem 'sinatra', '~> 1.4' gem 'wlang', '~> 2.3' -gem "alf-core", :git =>
"git://github.com/alf-tool/alf-core.git" -gem "alf-doc", :git =>
"git://github.com/alf-tool/alf-doc.git" -gem "alf-sql", :git =>
"git://github.com/alf-tool/alf-sql.git" -gem "alf-sequel", :git =>
"git://github.com/alf-tool/alf-sequel.git" -gem "alf-rack", :git =>
"git://github.com/alf-tool/alf-rack.git" +gem "alf-core", "~> 0.16.3"
+gem "alf-doc", "~> 0.16.3" +gem "alf-sql", "~> 0.16.3" +gem "alf-
sequel", "~> 0.16.3" +gem "alf-rack", "~> 0.16.3"
> @blambeau <https://github.com/blambeau> 21 commits.
>

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/alf-

tool/try-alf/pull/3#discussion_r555895254>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/AAAN4ZTZR7Y7W6PKE5SYVEDSZRYPLANCNFSM4V7ETIHA>.

File: Gemfile (hunk 1 above)
Comment (ciremoussadia) [reply Gemfile:3]: Just to know why don't you want to have the ruby version specified @blambeau ?

**Comment by blambeau** [comment]
Good question. I never use it, simply. I don't know why it exists, and

frequently apply the "don't use something you don't understand" rule.

On January 12, 2021, GitHub <notifications@github.com> wrote:
> @ciremoussadia commented on this pull request.
>

> In Gemfile <https://github.com/alf-tool/try-
> alf/pull/3#discussion_r555957451>:
>

> @@ -1,32 +1,26 @@ source "https://rubygems.org" -ruby '1.9.3' +ruby
'2.7.2'
> Just to know why don't you want to have the ruby version specified
> @blambeau <https://github.com/blambeau> ?
>

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/alf-

tool/try-alf/pull/3#discussion_r555957451>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/AAAN4ZTCMWPM2PMSOOITEHLSZSCZLANCNFSM4V7ETIHA>.

**Comment by ciremoussadia** [comment]
@blambeau It helps to handle dependencies by specifying the ruby version that the application needs. Certainly in this application, we don't need but in other cases, it's absolutely necessary.

**Comment by blambeau** [comment]
Do as you think is good. Please keep going.

On January 12, 2021, GitHub <notifications@github.com> wrote:
> @blambeau <https://github.com/blambeau> It helps to handle
> dependencies by specifying the ruby version that the application
> needs. Certainly in this application, we don't need but in other
> cases, it's absolutely necessary.
>

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/alf-

tool/try-alf/pull/3#issuecomment-758842530>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/AAAN4ZTUOPUR3SO424M2K6LSZSGK3ANCNFSM4V7ETIHA>.

**Pull request merged**